### ISSUE TITLE
TypeError [ERR_INVALID_CALLBACK]: Callback must be a function

### ISSUE THREAD

**Issue by svbatalov** [sehrope/node-pg-db#6]
On Node.js v9+ `pg-db` throws this error:
```
$ node
> db = require('pg-db')();
...
> db.query('SELECT NOW(), (err, rows) => console.log(err, rows) );
undefined
> TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at Object.randomBytes (internal/crypto/random.js:48:11)
    at Object.options.source (.../node_modules/rand-token/index.js:60:25)
    at generate (.../node_modules/rand-token/index.js:95:29)
    at .../node_modules/pg-db/lib/index.js:385:21
    at .../node_modules/pg-db/lib/index.js:428:18
    at bound (domain.js:396:14)
    at runBound (domain.js:409:12)
    at .../node_modules/pg/lib/pool.js:84:11
    at .../node_modules/generic-pool/lib/generic-pool.js:347:9
```

Looks like this is the bug in `node-rand-token` which was fixed in recent versions. Can you update the version of `node-rand-token` in `package.json`?

Related issues:
* https://github.com/nodejs/node/issues/16778
* https://github.com/sehrope/node-rand-token/issues/10

Thanks.

**Comment by sehrope** [comment]
Are you on the latest version of this module?

The rand-token dependency was updated a few months ago (https://github.com/sehrope/node-pg-db/commit/48f240160ed17ec735b2c8a89829098b032318a4) and published to NPM as pg-db v2.0.0.

**Comment by svbatalov** [comment]
You're right, version 2.0.0 works fine. Sorry for the noise! Thanks!
